CMakeLists.txt

```diff
@@ -30,6 +39,11 @@ catkin_package(
     DEPENDS Boost Eigen OMPL OpenRAVE
 )
 
+if(OR_OMPL_HAS_BOOSTSMARTPTRS)
+else(OR_OMPL_HAS_BOOSTSMARTPTRS)
+add_definitions(-std=c++11)
```

Comment:
Actually - you are using these flags in headers that may be included by other projects. Please use `configure_file` to generate a `.h` file with a `#cmakedefine` in it. That way the variable will be set for other projects that include `or_ompl`'s headers.